README.md

```diff
@@ -46,7 +47,34 @@ neumorphic.css is a lightweight, responsive and modern CSS framework.
 - [Copyright and License](#license)
 
 ## <a name="gettingstarted"></a>Getting started
-neumorphic.css is available via either NPM (preferred), Yarn, or a CDN.
+neumorphic.css is available via either direct installation, NPM, Yarn, or a CDN.
```

Comment:
The direct installation option should be after the CDN option and NPM should remain as the preferred option as many developers still prefer to use package managers on their projects.